src/components/books/BookSelect.tsx

```diff
@@ -51,7 +51,7 @@ const BookSelect = () => {
 				onSubmit={onSubmit}
 				value={currentValue}
 				setValue={setCurrentValue}
-				autoCompleteList={booksIds}
+				autocompleteList={booksIds}
```

Comment:
I don't think its necessary to rename to autoComplete, because its one word "autocomplete"